src/widgets/base.vue

```diff
@@ -35,8 +35,6 @@ export default {
 		isValid() {
 			const statefromStore = this.$store.state.vState[this.path].v
 			return statefromStore !== null ? statefromStore : true;
-
-
 			//return (this.$store.state.vState[this.path].e || []).length === 0
```

Comment:
this commented out line could be removed at this time as well, it is not in use anyhow.